Cargo.toml

```diff
@@ -10,10 +10,9 @@ homepage = "https://github.com/ctrlcctrlv/justify"
 
```

Comment:
Certainly you want to bump the version here, right? (Line 3.)

Reply:
I would leave releases up to you. :)

I can bump it if you want?

Reply:
Yes please, just like when committing to Rust libc, it's good etiquette